components/constellation/constellation.rs

```diff
@@ -1458,7 +1459,8 @@ impl<Message, LTF, STF> Constellation<Message, LTF, STF>
 
     fn handle_script_loaded_about_blank_in_iframe_msg(&mut self,
                                                       load_info: IFrameLoadInfo,
-                                                      layout_sender: IpcSender<LayoutControlMsg>) {
+                                                      layout_sender: IpcSender<LayoutControlMsg>,
+                                                      doc_type: DocumentType) {
```

Comment:
This is used for both initial and non-initial `about:blank` loads since they are both created on the script side.